backend/src/service/user/user_service.py

```diff
@@ -0,0 +1,80 @@
+import traceback
+from typing import List, Dict, Any, Optional
+
+from sqlalchemy.exc import SQLAlchemyError
+from werkzeug.security import generate_password_hash
+
+from common.logger import get_common_logger
+from models.user import UserModel
+
+
+class UserService:
+    def __init__(self, logger=None) -> None:
+        if logger is None:
+            logger = get_common_logger(__name__)
+        self.logger = logger
+
+    def create_user(self, data: Dict[str, str]) -> bool:
+        """
+        create new user
+        :param data: data dict includes name, email, address_id, password, role, country
+        :return: True if creation success else False
+        """
+        if not data.get("name") or not data.get("email") or not data.get("address_id") or not data.get("password") or not data.get("role") or not data.get("country"):
```

Comment:
It would be great if you could bubble up the validations error into the json response. I am thinking of something like this:

For success responses, response status is 2xx. Format is:
```
{
   "data": /*actual data*/
}
```

For error responses, status 4xx or 5xx. Format is:
```
{
  errors: [
    {
      error_code: "E0001",
      reason: "Validation Failed",
      description: "Name cannot be blank."
    }
  ]
}
```